### ISSUE TITLE
scrolling bitmapped images

### ISSUE THREAD

**Issue by videosmith** [Cuperino/QPrompt#49]
How do I insert a .jpg? I attempted drag and drop, the result only displayed the file's location text.

Also, when exporting as html, where are the graphics expected to be located when the script is re-imported?

**Device information**
 - Operating System: [Debian Bullseye 64bit RPi 4b]
 - QPrompt Version: [e.g. 1.0.0]

**Comment by Cuperino** [comment]
Yes. Although QPrompt supports displaying bitmaps in most mayor formats and SVG vector graphics, it currently provides no means of adding them. Reference to them must be added into an HTML file prior to opening it...

I wanted to avoid an issue of Imaginary Teleprompter's, which is that image data isn't always a part of saved files. So I wasn't going to add the means to add images until there were means to bundle these images with a save files. The other thing that held me back from adding this was that I would have to implement controls to manipulate these images, that worked with mouse and touch inputs as well. Most people don't use images in Imaginary Teleprompter, so I deemed it low priority.

If you were to place _bitmap support_ between two other features or fixes, ordered from less to more priority, which feature or fix would go before this, and which would go after?

**Comment by videosmith** [comment]
Low priority. As you said, will probably becused infrequently.
    On Monday, January 31, 2022, 09:09:29 PM EST, Javier O. Cordero Pérez ***@***.***> wrote:  
 
 


Yes. Although QPrompt supports displaying bitmaps in most mayor formats and SVG vector graphics, it currently provides no means of adding them. Reference to them must be added into an HTML file prior to opening it...

I wanted to avoid an issue of Imaginary Teleprompter's, which is that image data isn't always a part of saved files. So I wasn't going to add the means to add images until there were means to bundle these images with a save files. The other thing that held me back from adding this was that I would have to implement controls to manipulate these images, that worked with mouse and touch inputs as well. Most people don't use images in Imaginary Teleprompter, so I deemed it low priority.

If you were to place bitmap support between two other features or fixes, ordered from less to more priority, which feature or fix would go before this, and which would go after?

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
